recipes/libtiff/all/conanfile.py

```diff
@@ -96,11 +96,20 @@ def requirements(self):
         if self.options.get_safe("webp"):
             self.requires("libwebp/1.2.0")
 
+    def validate(self):
+        if self.options.get_safe("libdeflate") and not self.options.zlib:
+            raise ConanInvalidConfiguration("libtiff:libdeflate=True requires libtiff:zlib=True")
+
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        os.rename("tiff-" + self.version, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
 
     def _patch_sources(self):
+        # Rename the generated Findjbig.cmake and Findzstd.cmake to avoid case insensitive conflicts with FindJBIG.cmake and FindZSTD.cmake on Windows
+        tools.rename(os.path.join(self.build_folder, "Findjbig.cmake"),
+                     os.path.join(self.build_folder, "ConanFindjbig.cmake"))
+        tools.rename(os.path.join(self.build_folder, "Findzstd.cmake"),
```

Comment:
```suggestion
        if tools.Version(self.version) >= "4.1":
            tools.rename(os.path.join(self.build_folder, "Findzstd.cmake"),
```

Reply:
Thanks! I've added your suggestion but with the next line also indented extra.